randovania/games/prime2/exporter/joke_hints.py

```diff
@@ -43,4 +43,10 @@
     "Morphing and staying still may attract some flying friends.",
     "Amorbis really digs raves.",
     "Navigating Dark Aether with low health causes Agon-y.",
+    "Captain's log, Datecycle 07.014.2. We engaged an unknown Space Pirate frigate. Pursuit initiated.",
+    "Your valor honors the Luminoth... those we have lost, and those we can yet save.",
+    "*Hacker Voice*: I'm in",
```

Comment:
is there an echoes joke I'm missing here?

Reply:
Not particularly. I just found the combination of green text + doing the same action as with hacking via Hacker Mode in MP +  unexpected outcome for the action amusing.

Reply:
it doesn't feel great to mii